cmd/awscollector/main.go

```diff
@@ -92,6 +92,15 @@ func main() {
 // our default component list. Flags also need to be parsed before creating the config provider.
 func buildAndParseFlagSet(featgate *featuregate.Registry) (*flag.FlagSet, error) {
 	flagSet := config.Flags(featgate)
+
+	// TODO: remove after ADOT Collector v0.30.0 is released
+	if err := featgate.Set("pkg.translator.prometheus.NormalizeName", false); err != nil {
```

Comment:
how users can opt-in into using the normalized names?

Reply:
The instructions are in the linked README. Do you think we need to duplicate it here?

Reply:
I'm asking if it is possible to override this from the command line, since this is hard coded. is it?

Reply:
Yes it is possible. The feature gate flags are parsed later on in collector initialization. If a user provides `--featuregates=+pkg.translator.prometheus.NormalizeName` at startup it will be overridden.